apps/src/code-studio/pd/workshop_dashboard/components/workshop_management.jsx

```diff
@@ -59,6 +50,29 @@ export class WorkshopManagement extends React.Component {
     }
   }
 
+  use_daily_survey_route = () => {
+    let workshop_date = new Date(this.props.date);
+
+    let new_local_summer_and_teachercon =
+      workshop_date.getFullYear() >= 2018 &&
```

Comment:
Can we set this up to avoid having hardcoded dates, and rather uses the correct dates for the current year in an automatic or at least centralized fashion?

Reply:
Though it sounds like these are actually hardcoded dates, in which case it would be good to comment on what they mean and why they don't need to change.